src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs

```diff
@@ -61,8 +61,8 @@ private static string GetDefaultContentTypeName(HostWorkspaceServices workspaceS
 
             if (mefHostServices != null)
             {
-                var registry = mefHostServices.GetExports<IContentTypeRegistryService>().Single().Value;
-                return mefHostServices.GetExports<ILanguageService, ContentTypeLanguageMetadata>()
+                var exports = mefHostServices.GetExports<ILanguageService, ContentTypeLanguageMetadata>();
```

Comment:
Oh, I see. Too many long type names. My concern here would be if somebody exports their own language service with a content type metadata by accident that weird things might happen. Asking for the exports of the specific language service type would be a bit safer (and cheaper).